Week1/homework/4-log-the-string.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+let myString;
+myString = "Basak Sila Kaymak";
+console.log("The value of the string will be my name as I wrote it.");
+console.log(myString);
+myString = "Sila Kaymak"
+console.log('my second name and surname will be the result.');
+console.log(myString);
```

Comment:
👍

Reply:
Also, remember that strings are immutable. This means that whatever memory was allocated on line 3 i.e when you created a string `Basak Sila Kaymak` will be taken back by JavaScript after it executes line 6 where you reassign `myString` to `Sila Kaymak`